packages/blocks/src/code-block/code-model.ts

```diff
@@ -7,7 +7,6 @@ export class CodeBlockModel
 {
   static version = 1;
   flavour = 'affine:code' as const;
-  type = 'code' as const;
```

Comment:
In `BaseBlockModel`, the `type` field is required, removing it directly is unsafe. For example, it will break the format bar in the code block(I can fix it by switching to `flavour+type`).

<img width="556" alt="Screenshot 2023-01-27 at 2 03 00 AM" src="https://user-images.githubusercontent.com/18554747/214913735-e364bf72-21ce-4e88-a4e5-618918dbe29e.png">

<img width="356" alt="Screenshot 2023-01-27 at 2 05 17 AM" src="https://user-images.githubusercontent.com/18554747/214914416-cb8140b2-fe8f-463e-8240-c651423dead3.png">

<img width="589" alt="Screenshot 2023-01-27 at 2 06 37 AM" src="https://user-images.githubusercontent.com/18554747/214914622-81e7bbb2-c632-4b3e-a29d-b17908e6705b.png">

https://github.com/toeverything/blocksuite/blob/4f085f4cc4aa0f7ba436afd117efa9486b324b3f/packages/blocks/src/components/format-quick-bar/format-bar-node.ts#L82

Reply:
It's just a workaround in `BaseBlockModel`, only the `flavour` field should be required. I don't want the default `type` field actually, which should have been commented 🤦‍♂️